dev-python/pydotplus/pydotplus-2.0.2.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit distutils-r1
+
+DESCRIPTION="Improved version of the old pydot project"
+HOMEPAGE="http://pydotplus.readthedocs.org/"
+SRC_URI="mirror://pypi/p/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
```

Comment:
Ideally I would like to be able to try and run the tests of the package.

Now I notice that the python_test() phase finds no tests to run.

I tried changing the line to `${EPYTHON} -m unittest test || die` which also finds no tests. I tried to point unittest to the script directly (via `${EPYTHON} -m unittest test/pydot_unittest.py || die`) and got this http://bpaste.net/show/94697a5519f0 .

Lastly, I followed the advice from here https://wiki.gentoo.org/wiki/Project:Python/Tests and tred navigating to the workdir to run `unit2 discover`. That also found no tests to run.